.nomad/conf/www.conf

```diff
@@ -17,6 +17,7 @@ pm.max_children = 3
 catch_workers_output = yes
 php_admin_value[date.timezone] = America/New_York
 php_admin_value[session.save_handler] = redis
+php_admin_value[memory_limit] = 1G
```

Comment:
It's really using a ton of memory?

Reply:
Yes, they're big PDFs and it's not very efficient.